vhdbuilder/packer/install-dependencies.sh

```diff
@@ -1,4 +1,9 @@
 #!/bin/bash
+set -e -o pipefail
+
+echo "##### debug starts"
+cat /home/packer/provision_installs.sh
```

Comment:
This file is pretty static, why wouldn't we have it in build VHD?

If so, How about this
cat /home/packer/provision_installs.sh > /dev/null

and do the same for all the below files?